test/core/sup_tree_gear/gear_log_writer_test.exs

```diff
@@ -97,4 +97,14 @@ defmodule AntikytheraCore.GearLog.WriterTest do
     assert timer2 != timer1
     assert Path.wildcard(@dir <> "/testgear.log.*.gz") == []
   end
+
+  test "should get and set write_on_terminal?. it is ok if the logger does not exist" do
+    assert !Writer.write_on_terminal?(:testgear)
```

Comment:
(optional)
There is `refute` for negative assertion (it's just my preference).

```suggestion
    refute Writer.write_on_terminal?(:testgear)
```

Reply:
Thank you! 
I fixed at https://github.com/access-company/antikythera/pull/182/commits/af19757add51f01becb31c0ba423a8ac6c72965f .